docs/src/main/asciidoc/security-openid-connect-client-reference.adoc

```diff
@@ -725,6 +725,8 @@ quarkus.oidc-client.client-id=quarkus-app
 
 # This key is used to retrieve a secret from the map of credentials returned from CredentialsProvider
 quarkus.oidc-client.credentials.client-secret.provider.key=mysecret-key
+# this is the keyring provided to the CredentialsProvider when looking up the secret, set only if required by the CredentialsProvider implementation
```

Comment:
```suggestion
# This is the keyring provided to the CredentialsProvider when looking up the secret, set only if required by the CredentialsProvider implementation
```

Sorry about that, but I'm pretty sure our Doc or QE team would prefer to align it with capital `T` in the previous comment :-)

Reply:
Np! Makes sense to me - I should have noticed the other lines were capitalized